definitions/PowerType.dbd

```diff
@@ -14,8 +14,6 @@ int RegenInterruptTimeMS
 float RegenPeace
 int Field_1_13_0_28211_009?
 int Field_1_13_0_28211_010?
-int Field_1_13_2_30073_003? // Field_1_13_0_28211_009 probably
```

Comment:
What about the comment?

Reply:
Field order in 1.13.2 and 8.0 db2s are very similar, in contrast to 1.13.0 and 7.3.5. As such, I left out anything related to 1.13.0. The comment(s) are probably correct, but didn't take any chances.